modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -37,6 +37,12 @@ import (
 	"go.uber.org/zap"
 )
 
+// fs.FileInfo -- note that this contains only a basename, not a full path
+const StaticFileInfoKey = "caddyhttp.fileserver.info"
+
+// string -- actual path used by caddy
+const StaticFilePathKey = "caddyhttp.fileserver.path"
```

Comment:
These should go at the bottom of the file. Also, it should be `http.file_server.*`, see the reverse proxy which sets up similar variables. Actually for the proxy we're just setting it in the replacer directly instead of as a var. I'm not sure which way we should go, both have valid arguments and are equally as usable.

Reply:
Attempting to follow convention more closely in 06091370efa046954c86784fbba3b7b91e794ff0.